### PR TITLE
new routes for user-intent

### PR DESCRIPTION
**Pull request by invincibleJai**
Signed-off-by: invinciblejai <mailjai.vardhan@gmail.com>

### CODE REVIEW

File: src/app/user-intent/user-intent.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.launch-pad {
+    margin: 1% auto;
+    margin-bottom: 1%;
+}
```
Comment (arunkumars08) [inline src/app/user-intent/user-intent.component.scss]: New line

File: src/app/user-intent/user-intent.component.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+@Component({
+    selector: 'f8-user-intent',
+    templateUrl: './user-intent.component.html',
+    styleUrls: ['./user-intent.component.scss']
+})
+
+export class UserIntentComponent implements OnInit {
+    public stackUrl: string;
+    public apiData: any;
+    public gateway: any = {};
```
Comment (arunkumars08) [inline src/app/user-intent/user-intent.component.ts]: Is this being used somewhere?

File: src/app/user-intent/user-intent.module.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { TabsModule, AccordionModule } from 'ngx-bootstrap';
+
+import { UserIntentComponent } from './user-intent.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    HttpModule,
+    FormsModule,
+    ModalModule,
```
Comment (arunkumars08) [inline src/app/user-intent/user-intent.module.ts]: Is this required?

File: src/app/user-intent/user-intent.module.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    AccordionModule.forRoot(),
```
Comment (arunkumars08) [inline src/app/user-intent/user-intent.module.ts]: Is this required?

File: src/app/user-intent/user-intent.module.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    TabsModule.forRoot()
```
Comment (arunkumars08) [inline src/app/user-intent/user-intent.module.ts]: Is this required?

File: src/app/user-intent/user-intent.component.html [hunk 6]
```diff
@@ -0,0 +1 @@
+<p> user intent view!! </p>
```
Comment (arunkumars08) [inline src/app/user-intent/user-intent.component.html]: New Line

File: src/app/user-intent/user-intent.component.scss (hunk 1 above)
Comment (invincibleJai) [reply src/app/user-intent/user-intent.component.scss]: done

File: src/app/user-intent/user-intent.component.html (hunk 6 above)
Comment (invincibleJai) [reply src/app/user-intent/user-intent.component.html]: done

**Pull request merged**